.github/CODEOWNERS

```diff
@@ -82,7 +82,7 @@ assets/               @DataDog/agent-integrations
 /tcp_queue_length/                        @DataDog/container-integrations @DataDog/agent-integrations
 /tcp_queue_length/*.md                    @DataDog/container-integrations @DataDog/agent-integrations @DataDog/documentation
 /tcp_queue_length/manifest.json           @DataDog/container-integrations @DataDog/agent-integrations @DataDog/documentation
-/tenable/                                 @anshumgargdd @DataDog/agent-integrations
+/tenable/                                 @DataDog/agent-integrations
```

Comment:
```suggestion
```
I think we can just remove this line since agent integrations owns everything by default https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/customizing-your-repository/about-code-owners#example-of-a-codeowners-file